client/asset/eth/nodeclient_harness_test.go

```diff
@@ -716,17 +716,17 @@ func prepareTokenClients(t *testing.T) {
 		t.Fatalf("unexpected error while waiting to mine approval block: %v", err)
 	}
 
-	receipt1, err := waitForReceipt(ethClient, tx1)
+	_, err = waitForReceipt(ethClient, tx1)
 	if err != nil {
 		t.Fatal(err)
 	}
-	spew.Dump(receipt1)
+	// spew.Dump(receipt1)
```

Comment:
`waitForReceipt` also dumped the receipt, so they were dumped twice, and without any context. I'm personally not a fan of using `spew.Dump` the way we do in these tests, but if we do, we need to add some additional messaging to indicate what is being shown.